docs/index.rst

```diff
@@ -2,26 +2,106 @@ Horovod documentation
 =====================
 Horovod improves the speed, scale, and resource utilization of deep learning training.
 
+Get started
+-----------
+Choose your deep learning framework to learn how to get started with Horovod.
+
+.. raw:: html
+
+    <button class="accordion">TensorFlow</button>
+    <div class="panel">
+      <p>To use Horovod with TensorFlow on your laptop:
+         <ol>
+            <li><a href="https://www.open-mpi.org/faq/?category=building#easy-build">Install Open MPI 3.1.2 or 4.0.0</a>, or another MPI implementation. </li>
+            <li>Install the Horovod pip package: <code>pip install horovod</code></li>
+            <li>Read <a href="https://horovod.readthedocs.io/en/latest/tensorflow.html">Horovod with TensorFlow</a> for best practices and examples. </li>
+         </ol>
+         Or, use <a href="https://horovod.readthedocs.io/en/latest/gpus_include.html">Horovod on GPUs</a>, in <a href="https://horovod.readthedocs.io/en/latest/spark_include.html">Spark</a>, <a href="https://horovod.readthedocs.io/en/latest/docker_include.html">Docker</a>, <a href="https://github.com/sylabs/examples/tree/master/machinelearning/horovod">Singularity</a>, or Kubernetes (<a href="https://github.com/kubeflow/kubeflow/tree/master/kubeflow/mpi-job">Kubeflow</a>, <a href="https://github.com/kubeflow/mpi-operator/">MPI Operator</a>, <a href="https://github.com/helm/charts/tree/master/stable/horovod">Helm Chart</a>, and <a href="https://github.com/IBM/FfDL/tree/master/etc/examples/horovod/">FfDL</a>).
```

Comment:
Is it possible to use include to reduce duplication, like this one?

Reply:
Using .. include:: would be ideal, but it will get ignored because .. raw:: will parse everything as HTML

I think I could do it with JavaScript, but would have to put the shared content in a separate file and reference it. Would you prefer to make edits in 1 place in a separate .js file, or make them 4+ times in the same .rst file? 

Another option is to change it from an accordion to buttons for each, and include the install steps in the individual Keras/PyTorch .rst files and use include to share the steps there

Reply:
I see, maybe let's keep it as-is for now.  Separate JavaScript file sounds a bit too magical :-)

Reply:
Sounds good :) You have good tech writing instincts!